web_last_viewed_records/static/src/js/main.js

```diff
@@ -0,0 +1,178 @@
+openerp.web_last_viewed_records = function(instance){
```

Comment:
I think you should test if the localStorage object exists here and continue only if so